USAGE.md

```diff
@@ -92,21 +92,28 @@ Descent 3 has two types of base directories:
 - The writable base directory can contain both read-write files and read-only files. There is only one writeable base directory. By default, the writable base directory gets set to the current working directory.
 - The read-only base directories can only contain read-only files. There can be any number of read-only base directories. By default, Descent 3 uses zero read-only base directories.
 
-You can set the writable base directory and the list of read-only base directories using the `-setdir`, `-useexedir` and `-additionaldir` command-line options (see [the next section](#command-line-options)).
+You can set the writable base directory and the list of read-only base directories using the `-setdir`, `-useexedir` and `-additionaldir` command-line options (see [the next section](#command-line-options)). Descent 3 also has a list of default read-only base directories. Normally, the list of default read-only base directories is empty, but you can change it by using the `DEFAULT_ADDITIONAL_DIRS` CMake option when compiling Descent 3 (see [BUILD.md’s Build Options section](./BUILD.md#build-options)).
 
 When Descent 3 tries to find a read-only file, then it will look through the list of base directories in this order:
 
 - the last read-only base directory that was specified on the command-line,
 - the second-to-last read-only base directory that was specified on the command-line,
 - the third-to-last read-only base directory that was specified on the command-line,
 - …
-- the first read-only base directory that was specified on the command-line and, finally,
+- the first read-only base directory that was specified on the command-line,
+- the last read-only base directory from the `DEFAULT_ADDITIONAL_DIRS` list\*,
+- the second-to-last read-only base directory from the `DEFAULT_ADDITIONAL_DIRS` list\*,
+- the third-to-last read-only base directory from the `DEFAULT_ADDITIONAL_DIRS` list\*,
+- …
+- the first read-only base directory from the `DEFAULT_ADDITIONAL_DIRS` list\*, and, finally,
 - the writable base directory.
 
+<sub>\*These ones are skipped if you use the `-nodefaultadditionaldirs` command-line option.</sub>
```

Comment:
as `DEFAULT_ADDITIONAL_DIRS` has lower priority than `-additionaldir`, I don't think this is very useful